diptest/game_test.go

```diff
@@ -1039,3 +1053,273 @@ func TestTrueSkillLinksFromFinishedGames(t *testing.T) {
 
 	})
 }
+
+func TestWithoutMustering(t *testing.T) {
+	gameDesc := String("test-game")
+	env1 := NewEnv().SetUID(String("uid"))
+	gameID := env1.GetRoute(game.IndexRoute).Success().
+		Follow("create-game", "Links").Body(map[string]interface{}{
+		"Variant":            "France vs Austria",
+		"NoMerge":            true,
+		"Desc":               gameDesc,
+		"Private":            false,
+		"SkipMusterPhase":    true,
```

Comment:
Possibly this should be the default?

Reply:
Really? I felt that it was better to have mustering as default?

Reply:
I don't have a strong opinion about mustering for real games, but for tests then I think we'll want to skip mustering far more than not. Anyway, no problem either way from me.

Reply:
Sure, that'd save some time.

I won't fix it now though, since this entire PR is on ice...